test/infrastructure/scripts/step-01-config.sh

```diff
@@ -1,4 +1,5 @@
 export CLUSTER_NAME="${CLUSTER_NAME:-karpenter-test-cluster}"
+export K8S_VERSION="${K8S_VERSION:-1.22}"
```

Comment:
IMO we should just take an opinion directly in the eksctl config. Same with region and karp version.